db/migrate/20221129094342_create_value_definitions.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+# Data model for storing value definitions
+class CreateValueDefinitions < ActiveRecord::Migration[7.0]
+  def change
+    create_table :values, id: :uuid do |t|
```

Comment:
I think we'll need a column in this table for the `value` itself and the column type would be string. Unless @skateman told you otherwise?